src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPageSpec.groovy

```diff
@@ -28,30 +41,68 @@ class LoginPageSpec extends SpringContextAwareSpecification {
         tester = new WicketTester(wicketApplication)
     }
 
-    def "Should redirect to Admin Dashboard after successful login"() {
+    Should "Redirect to Admin Dashboard after successful login"() {
```

Comment:
Convention is to have tests named as "Should do something" it is much easier to read test reports and generally it improves readability.